docs/07-BDD.md

```diff
@@ -259,11 +259,11 @@ Given I am logged in as 1
 
 Steps are defined in Context files. Default context is an actor class, i.e. for acceptance testing suite default context is `AcceptanceTester` class. However, you can define steps in any classes and include them as contexts. It is useful to define steps in StepObject and PageObject classes.
 
-To list all defined steps run `gherkin:steps` command:
+To list all defined steps run `gherkin:steps acceptance` command:
```

Comment:
I think that this sentence should make it clear that parameter is the name of suite, specific example for acceptance suite is given below.

In the same manner as snippets description above, it could be something like:

> To list all defined steps run `gherkin:steps` command for the given suite:
```sh
codecept gherkin:steps acceptance
```